service/controller/app/v1/resource/chartoperator/create.go

```diff
@@ -0,0 +1,46 @@
+package chartoperator
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/app-operator/service/controller/app/v1/controllercontext"
+	"github.com/giantswarm/app-operator/service/controller/app/v1/key"
+)
+
+func (r Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	cr, err := key.ToCustomResource(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+	cc, err := controllercontext.FromContext(ctx)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	// Check whether TC has chart-operator deployment yet.
```

Comment:
```suggestion
	// Check whether tenant cluster has a chart-operator helm release yet.
```

Nit but this is more accurate I think as we are using helmclient to interact with the cluster.